include/eos/fitting/ceres_nonlinear.hpp

```diff
@@ -145,35 +166,35 @@ struct LandmarkCost
         using namespace glm;
         // Generate shape instance (of only one vertex id!) using current parameters and 10 shape
         // coefficients: Note: Why are we not returning a glm::tvec3<T>?
-        const auto point_arr =
-            get_shape_point<T>(shape_model, blendshapes, vertex_id, shape_coeffs, blendshape_coeffs);
+        const auto point_arr = get_shape_point<T>(shape_model, blendshapes, vertex_id, shape_coeffs,
+                                                  blendshape_coeffs, get_num_cam_params(use_perspective));
 
         // Project the point to 2D:
-        const tvec3<T> point_3d(point_arr[0], point_arr[1], point_arr[2]);
+        const auto point_3d = tvec3<T>(point_arr[0], point_arr[1], point_arr[2]);
```

Comment:
I like to use `auto` in many scenarios but here when initialising variables in my opinion it just makes the code a bit harder to read. To be consistent with the style of the library I'd leave the type on the left side here and then no auto is needed. (also the same applies to the few lines below)

Reply:
OK. Will change it.